packages/Demo/Entity/RectorRun.php

```diff
@@ -18,10 +18,10 @@
 use Rector\Website\Exception\ShouldNotHappenException;
 use Symfony\Bridge\Doctrine\IdGenerator\UuidGenerator;
 use Symfony\Component\Uid\Uuid;
-use Symplify\SymplifyKernel\Strings\StringsConverter;
+use Symplify\PackageBuilder\Strings\StringFormatConverter;
 
 #[Entity]
-class RectorRun implements TimestampableInterface
+final class RectorRun implements TimestampableInterface
```

Comment:
This will probably fail in Doctrine.

Reply:
Reverted https://github.com/rectorphp/getrector.org/commit/896cd5908a50fca9f269db38e6b80cd517e9b6ff